CHANGES.txt

```diff
@@ -1,6 +1,12 @@
 Comtypes CHANGELOG
 ==================
 
+Release 1.1.11
+-------------
+ * Fix Python 3 compatibility. Thanks @kdschlosser.
+ * Fix `tlib_mtime` in the absence of typelib.
+ * Fix 64bit inproc server.
```

Comment:
Maybe we want to rephrase these, suggestions appreciated.

Reply:
I'm not sure what these refer to. I believe they date back to Thomas Heller.